editor/src/core/tailwind/tailwind-config.ts

```diff
@@ -0,0 +1,49 @@
+import { RevisionsState, TextFile, textFile, textFileContents } from '../shared/project-file-types'
+import { lintAndParse } from '../workers/parser-printer/parser-printer'
+
+export const PostCSSPath = '/postcss.config.js'
+export const TailwindConfigPath = '/tailwind.config.js'
+
+export const TailwindCSSVersion = '2.2.4'
+export const PostCSSVersion = '8.3.5'
+
+const TailwindConfigJs = `// tailwind.config.js
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {},
+  plugins: [],
+}`
+
+const PostCSSConfigJs = `// postcss.config.js
+module.exports = {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
```

Comment:
We should either remove `autoprefixer` from this or add it to the list of dependencies too (I think we might have to go with the latter, I haven't checked if the default tailwind set up works without autoprefixer)

Reply:
fixed